src/rpc/fees.cpp

```diff
@@ -94,6 +96,76 @@ static RPCHelpMan estimatesmartfee()
     };
 }
 
+static RPCHelpMan estimatefee()
+{
+    return RPCHelpMan{
+        "estimatefee",
+        "\nEstimates the approximate fee per kilobyte needed for a transaction to begin\n"
```

Comment:
This is confusing I think.  It's titled `estimatefee` however a fee is usually a bitcoin amount.  A feerate however is a bitcoin amount per size.

Reply:
should this be "estimatefeerate"?

Reply:
I will update this here #31664

Reply:
The new RPC is gone; I've integrated this new feature into `estimatesmartfee` and maintained backward compatibility for users who want only block policy, using a flag.